src/Client/AcsfClient.php

```diff
@@ -63,7 +63,7 @@ public function ping(): bool {
    *   All the existing site the user has permission to view.
    */
   public function listSites(): array {
-    $response = $this->getJsonResponseBody($this->get('sites'), 'No sites found.');
+    $response = $this->getJsonResponseBody($this->get('sites?limit=1000'), 'No sites found.');
```

Comment:
Let's include $limit as function param with default value and use it as a query param.